### ISSUE TITLE
Dropdown menus in nav should be dismissible (SC 1.4.13 Level AA)

### ISSUE THREAD

**Issue by jeremy-w** [jeremy-w/orthic#21]
https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html
